includes/scrub-options.php

```diff
@@ -91,6 +91,9 @@ function( $v ) use ( $modules_to_disable ) {
 	}
 
 	update_option( 'safety_net_options_scrubbed', true );
+
```

Comment:
Similar to the backup options, I left this call to `update_option()` in place, because we're creating this option, so it's just as (if not more) likely  to be adding the option instead of updating it.